app/javascript/lib/rich_text/unfurl/unfurler.js

```diff
@@ -44,6 +46,38 @@ export default class Unfurler {
     }
   }
 
+  #didChange(event) {
+    this.#debouncedCheckForUrls(event)
+  }
```

Comment:
Maybe there is a better name for #didChange that communicates what it does better? E.g. `checkForUrls` or `unfurl`?